spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/mapping/CommonMappingRuleInterpreter.scala

```diff
@@ -66,13 +65,9 @@ trait CommonMappingRuleInterpreter {
 
     val mappingTableDef = dao.getMappingTable(rule.mappingTable, rule.mappingTableVersion)
 
-    val ruleFilter = if (rule.mappingTableFilter.nonEmpty) {
-      rule.mappingTableFilter
-    } else {
-      // This is a workaround until UI supports filter definition. Until then, the filters can be set via configuration.
-      FilterFromConfig.loadFilter(rule.mappingTable)
-    }
+    val ruleFilter = rule.mappingTableFilter
```

Comment:
I think even this `val` can go.

Reply:
Well, it could go and one would just use `rule.mappingTableFilter`. However, my point here is that it improves the readability of the code. I deliberately left it as `ruleFilter` and `mappingTableFitler`, those two named filters that will get merged later:

```scala
    (ruleFilter, mappingTableFilter) match {
      case (Some(a), Some(b)) => Option(a and b)
      case (Some(a), None)    => Option(a)
      case (None, Some(a))    => Option(a)
      case (None, None)       => None
```

In my opinion, it conveys the meaning better than if the `val` gets removed. So here, I'd really like to keep it as-is if this perspective makes sense to you.

Reply:
Fair enough 👍